tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -23,14 +23,14 @@ class CompletionArgSuite extends BaseCompletionSuite {
   check(
     "arg-newline",
     s"""|object Main {
-        |  def foo(banana: String, apple: String) = ???
+        |  def foo(banana: String, tomato: String) = ???
```

Comment:
After making a deduplication, this tests started receiving an additional `apple` package symbol from jdk so I decided just to rename it